quack_app/lib/screens/homepage/home_screen.dart

```diff
@@ -0,0 +1,125 @@
+import 'dart:ui';
+
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:quack_app/components/navbar.dart';
+import 'package:quack_app/constants/constants.dart';
+import 'package:quack_app/core/menu_data.dart';
+
+class HomeScreen extends StatefulWidget {
+  final String title;
+  const HomeScreen({Key? key, required this.title}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  late MenuData menuData;
+  @override
+  void initState() {
+    // Re-cache updated values on view
+    super.initState();
+    menuData = MenuData();
+  }
+
+  void loadData() async {
+    await menuData.getCurrentlyServing();
+    await menuData.getTodaysMenu();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+        child: Column(
+      mainAxisAlignment: MainAxisAlignment.start, //
+      children: <Widget>[
+        //
```

Comment:
Is the // intentional?

Reply:
Yes, the auto formatting in VSCode formats it all into the same line unless theres a // there to keep it on separate lines